shared/actions/tracker.js

```diff
@@ -97,13 +98,22 @@ export function registerTrackerIncomingRpcs (): TrackerActionCreator {
 
 export function getProfile (username: string): TrackerActionCreator {
   return (dispatch, getState) => {
+    dispatch({type: Constants.updateUsername, payload: {username}})
```

Comment:
we should have had this in the first place, ensure the tracker store is ready for data